internal/server/v0/triple/golden/get_triples/observation.json

```diff
@@ -67,39 +67,5 @@
       "objectName": "Total Population",
       "provenanceId": "dc/base/CensusPEP"
     }
-  ],
-  "dc/o/kyv7dxe4s18eh": [
-    {
-      "subjectId": "dc/o/kyv7dxe4s18eh",
```

Comment:
@n-h-diaz : The observation node id seems to have changed. Prod version(dc_kg_2024_12_07_07_45_51) is having dc/o/kyv7dxe4s1 whereas the autopush version (dc_kg_2025_01_03_23_15_25) is having dc/o/48rnjqg9212e1. Is this expected?